tests/functional/generate_pages.py

```diff
@@ -840,6 +840,17 @@ def process_block(
                 page_filename=f'{repeating_block["id"]}-repeating-block.page.js',
             )
 
+    if block["type"] == "ListCollectorContent":
```

Comment:
you could do `if block["type"] in {"ListCollectorContent", "ListCollector"}: ...`
and then you can delete the for loop just above this one as they are duplicates

Reply:
What about list operations?
```python
list_operations = ["add", "edit", "remove"]
```
we don't want them for `ListCollectorContent`...

Reply:
yep exactly, so you'd leave that part alone, but iterate over the repeating blocks for either type of list collector, so end result would be
```
if block["type"] == "ListCollector":
    list_operations = ["add", "edit", "remove"]
    for list_operation in list_operations:
        process_block(
            block[f"{list_operation}_block"],
            dir_out,
            schema_data,
            spec_file,
            relative_require,
            page_filename=f'{block["id"]}-{list_operation}.page.js',
        )

if block["type"] in {"ListCollectorContent", "ListCollector"}:
    for repeating_block in block.get("repeating_blocks", []):
        process_block(
            repeating_block,
            dir_out,
            schema_data,
            spec_file,
            relative_require,
            page_filename=f'{repeating_block["id"]}-repeating-block.page.js',
        )
```

Reply:
Ok, understood.